frontend/public/components/utils/dropdown.jsx

```diff
@@ -319,6 +325,7 @@ export class Dropdown extends DropdownMixin {
     };
 
     _.each(items, (v, k) => addItem(k, v));
+    rows.sort(compareRows);
```

Comment:
You can use [`_.sortBy()` here](https://lodash.com/docs/4.17.11#sortBy).

Reply:
fixed, thanks